scss/components/_modal.scss

```diff
@@ -82,27 +82,25 @@
       }
 
       // Close icon
-      @if $enable-classes {
-        .close {
-          display: block;
-          width: 1rem;
-          height: 1rem;
-          margin-top: calc(var(#{$css-var-prefix}spacing) * -1);
-          margin-bottom: var(#{$css-var-prefix}spacing);
-          margin-left: auto;
-          background-image: var(#{$css-var-prefix}icon-close);
-          background-position: center;
-          background-size: auto 1rem;
-          background-repeat: no-repeat;
-          opacity: 0.5;
-
-          @if $enable-transitions {
-            transition: opacity var(#{$css-var-prefix}transition);
-          }
+      #{if($enable-classes, ".close", "a[rel='prev']")} {
```

Comment:
I was also thinking of replacing the regular version.
But it's actually better to keep the class so that we could do it like this:

```SCSS
#{if($enable-classes, ".close, a[rel='prev']", "a[rel='prev']")} {
````

Also, there are another one, lines [L62-L68](https://github.com/picocss/pico/blob/v2/scss/components/_modal.scss#L62-L68)
```SCSS
@if $enable-classes {
  .close {
    ...
  }
}
```

Reply:
> @asbjornu, I was also thinking of replacing the regular version. But it's actually better — not a breaking change anymore — to keep the class. 

It could be considered a breaking change to introduce default styling for `a[rel=prev]`, but I agree that this is so specific that the risk of breakage is virtually non-existent.

> We could do it like this:
> 
> ```css-scss
> #{if($enable-classes, ".close, a[rel='prev']", "a[rel='prev']")} {
> ```

Sure, I'll change it.

> Also, there are another one, lines [L62-L68](https://github.com/picocss/pico/blob/v2/scss/components/_modal.scss#L62-L68)

I'll include them as well and rebase back to `master`.